### ISSUE TITLE
Player Mechanic - Shooting/firing 

### ISSUE THREAD

**Issue by NimNimR4r** [TeaylThorn/HighwynnGame#1]
**Describe the Bug**
One of the major issues players had was only being able to shoot one way (to the right) and not being able to deal damage
Players also found that after a certain amount of time, the projectiles that were used as a place holder for the "dragon fire" wouldn't disappear in scene and would build up over the course of the game-play

***

**Version**
Version One 

***

**To Reproduce**
To reproduce what happened all the player would have to do would be to: 
1. Try and shoot at an enemy (in either direction)
2. Press the "attack" key **X**
3. The projectiles would only go one way (to the right of the player)

***

**Expected Behaviour**
Players were expecting to have dealt some kind of damage to enemies, or to have at least the attack mechanism working in both directions instead of just being able to shoot to the right 

***
**Additional Content**
* Players would like to be able to deal damage to the enemies that are in the game level
* To be able to attack whilst running 
* And to have that attack disappear after a certain time instead of building up in scene... 

***

**TLDR**
* Player firing at enemies or certain objects needs to be updated so that they can "shoot" both ways  (instead of just to the right)
* Having the fire being able to deal damage to enemies
* Having the fire/"bullets" disappear after a certain time as well
